docs/src/componentDocs/Spacer/examples/SpacerWithPixelExample.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { Spacer } from '@brightlayer-ui/react-components';
+
+export const SpacerWithPixelExample = (): JSX.Element => (
+    <Box>
+        <Box sx={{ width: '300px', height: '56px', display: 'flex' }}>
+            <Spacer maxWidth={25} style={{ background: '#4da3d4' }}>
+                25
+            </Spacer>
+            <Spacer maxWidth={75} style={{ background: '#f5db6d' }}>
+                75
+            </Spacer>
+            <Spacer maxWidth={200} style={{ background: '#da7777' }}>
```

Comment:
Sorry I just noticed this. I think you should be using `width` instead of `maxWidth` here.

Reply:
updated